apps/src/code-studio/clientState.js

```diff
@@ -2,6 +2,7 @@
  * @file Helper functions for accessing client state. This state is stored in a
  *       combination of cookies and HTML5 web storage.
  */
+import { safelySetItem } from '../utils';
```

Comment:
I know you're just moving this already-existing function, but now might be a good time to also give it a better name; we already have `trySetLocalStorage` in utils, might be nice to have this one similarly named `trySetSesstionStorage`